app/views/internal/broadcasts/show.html.erb

```diff
@@ -21,7 +21,7 @@
       <% if @broadcast.processed_html %>
         <div>
           </p><strong>Preview</strong></p>
-          <div class="broadcast-wrapper visible <%= banner_class(@broadcast) %>">
+          <div class="visible <%= banner_class(@broadcast) %>">
```

Comment:
This is the right direction, but not _quite_ right. We don't want to remove `broadcast-wrapper`, because that's the style used throughout the site [when the broadcast is displayed](https://github.com/thepracticaldev/dev.to/blob/232c67dd0a50be92bab14bed9b76328d013c6d0a/app/views/shell/_top.html.erb#L61). We need that class because we want the preview on the admin side to have the same styling as what will be used throughout the site.

The real culprit here is `display: none;`, not the whole `broadcast-wrapper` class. You'll need to override the `display` property within the internal stylesheets (we're already doing something very similar to that).

Reply:
Gotcha! I've made this change.

Reply:
Here are the styles for `broadcast-wrapper` that apply _only_ to the `/internal` pages: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/internal/layout.scss#L54-L57

If you add an explicit `display` property here, you should get the result that you want